internal/testutil/acc/database_user.go

```diff
@@ -1,130 +1,54 @@
 package acc
 
 import (
```

Comment:
Any reason for not moving all these methods to the `databaseuser_test` package? Seems like they are only used there

Reply:
sorry at the end I think i'm not going to do this refactor in this PR as it's not related to the main topic of the PR and will increase changes still more